phac-nml/irida
# Fix add group member search

## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Fixed search when adding a user to a group to be case insensitive.

**To test this change:**

First create a few users:
1) User1: username: test01, first_name: tEst01, last_name: User
2) User2: username: test02, first_name: test02, last_name: user
3) User3: username: test03, first_name: TEST, last_name: USER

Then create a group and then visit the `Groups` page. Click on the group name that you have created. Add a member by clicking the `Add Group Member` button. In the search type in "TE", then "te", and then "Te". All three searches should return all 3 users each time.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

Fixes #317 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] User documentation updated for UI or technical changes.~~
